### PR TITLE
Change dialogs in onboarding flow (EXPOSUREAPP-13993)

### PR DESCRIPTION
**Pull request by SamuraiKek**
Updating the dialogs in the onboarding flow to the new dialog template.

### CODE REVIEW

File: Corona-Warn-App/src/main/res/navigation/nav_graph_onboarding.xml [hunk 1]
```diff
@@ -47,6 +38,15 @@
...
     </fragment>
+    <fragment
+        android:id="@+id/onboardingTracingFragment"
+        android:name="de.rki.coronawarnapp.ui.onboarding.OnboardingTracingFragment"
+        android:label="OnboardingTracingFragment"
+        tools:layout="@layout/fragment_onboarding_tracing">
+        <action
+            android:id="@+id/action_onboardingTracingFragment_to_onboardingTestFragment"
+            app:destination="@id/onboardingTestFragment" />
+    </fragment>
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/navigation/nav_graph_onboarding.xml:49]: how is it related to dialog change ?

Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/navigation/nav_graph_onboarding.xml:49]: I just rearanged the fragments order in the file so we have better readability. Each fragment in the flow is now one after the other in here as well.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/onboarding/OnboardingTracingFragment.kt [hunk 2]
```diff
@@ -76,12 +70,13 @@ class OnboardingTracingFragment : Fragment(R.layout.fragment_onboarding_tracing)
         binding.onboardingTracingContainer.sendAccessibilityEvent(AccessibilityEvent.TYPE_ANNOUNCEMENT)
     }
 
+    // TODO update deprecated fun with registerForActivityResult() method
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/onboarding/OnboardingTracingFragment.kt:73]: it is reported by sonar https://sonarcloud.io/project/issues?id=corona-warn-app_cwa-app-android&pullRequest=5597&resolved=false&types=CODE_SMELL

Comment (SamuraiKek) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/onboarding/OnboardingTracingFragment.kt:73]: Ok I'll remove the comment and I'll try to open a task in Jira.

Comment (mtwalli) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/onboarding/OnboardingTracingFragment.kt]: Sounds good, I'm not sure if that is possible with ENS dialogs. I already reported an issue to Google some time ago ,but no updates yet https://issuetracker.google.com/issues/180199984

**Review by mtwalli — APPROVED** [review]

**Pull request merged**